pwd-service/src/main/java/org/pwd/web/download/DownloadController.java

```diff
@@ -46,28 +53,47 @@ public String showTemplates(Model model) {
     @RequestMapping(value = "/{templateName}/{cmsName}", method = RequestMethod.GET)
     public String showTemplateDownloadForm(@PathVariable String templateName, @PathVariable String cmsName, Model model) {
 
-        model.addAttribute("template", Template.valueOf(templateName));
-        model.addAttribute("cms", cmsName);
+        template = Template.valueOf(templateName);
```

Comment:
prawie dobrze, ale nie wydaje Ci sie ze to bez sensu, odbieranie czegos z requesteu i wrzucanie tego do modelu?:)

zrob przez geta ?templateName=x&cms=wordpress
i potem w twigu masz do tego dostep przez {{ request.getParameter('cms') }}